examples/adjoints/adjoints_ex5/adjoints_ex5.C

```diff
@@ -740,15 +745,15 @@ int main (int argc, char ** argv)
       // 32-bit floats
       if(param.timesolver_tolerance)
       {
-        if(std::abs(system.time - (1.00285)) >= 2.e-4)
+        if(std::abs(system.time - (1.0089)) >= 2.e-4)
         libmesh_error_msg("Mismatch in end time reached by adaptive timestepper!");
 
-        if(std::abs(sensitivity_0_0 - (-4.85298)) >= 2.e-4)
+        if(std::abs(total_sensitivity - 4.87767) >= 2.e-4)
         libmesh_error_msg("Mismatch in sensitivity gold value!");
       }
       else
       {
-        if(std::abs(sensitivity_0_0 - (-4.83551)) >= 2.e-4)
+        if(std::abs(total_sensitivity - 4.83551) >= 2.e-4)
```

Comment:
Whoa, what happened here?  Our gold sensitivities just did a sign flip?

Reply:
Yeah, I had the wrong sign earlier.

Reply:
How'd we not catch it in CI?

Reply:
I meant I had the wrong sign in the analytic calculation.

Reply:
Sure, but how did we not catch that in CI before?  We've got two timesolver_tolerance=1.0 cases in run.sh.  Those are supposed to hit these tests.  If they disagree with the values in these tests they're supposed to die screaming.  If the values were wrong but the numerics were close to right, how did they not die screaming?

Reply:
Two things were wrong, the sign on the analytic calculation as well a local reimplementation of adjoint_qoi_parameter_sensitivity we were using to numerically evaluate the sensitivity.

I switched to the library's sensitivity method for the integrate_adjoint_sensitivity definition, saw the sign was different, went back and rechecked my analytic calculation, and realized the sign there had to be positive as well.

Reply:
Ah, I got it.  The same sign error was in the original adjoints_ex5.C code?